src/services/postVideoToPlaylist.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const postVideoToPlaylist = (authToken, playlistId, video) =>
+	axios.post(
+		`/api/user/playlists/${playlistId}`,
+		{ video },
+		{ headers: { authorization: authToken } }
+	);
+
+export { postVideoToPlaylist };
```

Comment:
Suggestion: You can keep all these functions in one file rather than separate files for each functions.

Reply:
Thank you for your suggestion. I prefer separating functions out because all these functions perform separate API calls.